larray/core/convert.py

```diff
@@ -0,0 +1,229 @@
+from __future__ import absolute_import, print_function
```

Comment:
I consider that df_asslarray is not an "IO" function. But OK, I can let df_aslarray in array.py